spec/spec_helper.rb

```diff
@@ -34,6 +34,7 @@
   params(
     pack_name: String,
     dependencies: T::Array[String],
+    violations: T::Array[String],
```

Comment:
Based on [this line](https://github.com/rubyatscale/parse_packwerk/pull/35/files#diff-d01840d7b2ab0f3ab0269a58355a437cee1a8eb933f46fbbb6fd4653be8ffbabR14), I think this should be

```suggestion
    violations: T::Array[ParsePackwerk::Violation],
```

Reply:
Thanks for catching this.